src/Message/Formatter.php

```diff
@@ -222,14 +233,16 @@ private function getContext(Message $message, array $commonContext): string
      */
     private function getTrace(Message $message): string
     {
-        $traces = (array) $message->context('trace', []);
+        /** @psalm-var Backtrace $traces */
+        $traces = $message->context('trace', []);
 
         foreach ($traces as $key => $trace) {
             if (isset($trace['file'], $trace['line'])) {
                 $traces[$key] = "in {$trace['file']}:{$trace['line']}";
             }
         }
 
+        /** @var string[] $traces */
```

Comment:
Element `line` can be int.

Reply:
On this line `$traces` can contain a string or an array (Backtrace)
```suggestion
       /** @var string[]|Stringable[] $traces */
```

Reply:
Oh, yes. This is need little refactoring. I make this in other PR.